app/models/list.rb

```diff
@@ -38,7 +41,7 @@ def as_json(*)
       only: [:id, :name],
       include: [:items]
       }).merge({
-        unbought_count: items.unbought.count,
+        unbought_and_active: items.unbought_and_active.count,
```

Comment:
This feels like a smell.  Is there a circumstance on the frontend where we would ever want the count of unbought and inactive items?  If there isn't, it seems like this should just be `unbought_count` as the key, and let the "and active" be implicit rather than part of the name.

Reply:
You are correct there most likely won't be an instance where we would ever need the count of unbought and inactive items. The variable name was purely for clarity and definition reasons. Good suggestion.

Reply:
Fixed: https://github.com/Brian-Demon/shopping-list/pull/69/commits/e1d4c2c04a98b88bf35a4bbb26218f98a1c2f16f